README.md

```diff
@@ -1,11 +1,53 @@
 # Tektoncd Operator
 
-Kubernetes [Operator](https://operatorhub.io/getting-started) to
-manage installations, updates and removal of Tektoncd projects
-(pipeline, dashboard, …)
-
-> The goal of an Operator is to put operational knowledge into
-> software. Previously this knowledge only resided in the minds of
-> administrators, various combinations of shell scripts or automation
-> software like Ansible. It was outside of your Kubernetes cluster and
-> hard to integrate. With Operators, we changed that.
+Kubernetes [Operator](https://operatorhub.io/getting-started) to manage installations, updates and removal of Tektoncd projects (pipeline, dashboard, …)
+
+The following steps will install [Tekton Pipeline](https://github.com/tektoncd/pipeline) and configure it appropriately for your cluster.
+1. Create namespace: `tekton-operator`  
+
+    `kubectl create namespace tekton-operator`
+    
+2. Apply Operator crd  
+
+    `kubectl apply -f deploy/crds/*_crd.yaml`
+    
+3. Deploy the Operator  
+
+    `kubectl -n tekton-operator apply -f deploy/`  
+    
+    The Operator will automatic install `Tekton pipeline` with `v0.5.2` in the namespace `tekton-pipeline`
+
+## Development Prerequisites
+1. [`go`](https://golang.org/doc/install): The language Tektoncd-pipeline-operator is
+   built in
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
+   interacting with your kube cluster
+1. operator-sdk: https://github.com/operator-framework/operator-sdk
+
+## Building the Operator Image
+1. Enable go mod  
+
+    `export GO111MODULE=on`
+    
+2. Build go and the container image  
+
+    `operator-sdk build ${YOUR_REGISTORY}/openshift-pipelines-operator:${IMAGE_TAG}`
+    
+3. Push the container image  
+
+    `docker push ${YOUR-REGISTORY}/openshift-pipelines-operator:${IMAGE-TAG}`
+    
+4. Edit the 'image' value in deploy/operator.yaml to match to your image  
+
+## The CRD
+This is a sample of [crd](https://github.com/tektoncd/operator/blob/master/deploy/crds/operator_v1alpha1_config_cr.yaml)
```

Comment:
I remember we have a demo saying we can either go with the auto-install or manual install.
Shall we add the instruction on it?